{"uuid":"93e6422b-d93f-45a5-afb2-d51c92cdb1a5","historyId":"4b699603eb3a12b293e1cebfdc70d431","fullName":"api.tests.CreateUserTests.deleteUserTest","labels":[{"name":"package","value":"api.tests.CreateUserTests"},{"name":"testClass","value":"api.tests.CreateUserTests"},{"name":"testMethod","value":"deleteUserTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"p.project_rb"},{"name":"subSuite","value":"api.tests.CreateUserTests"},{"name":"host","value":"LAPTOP-14OH4E5T"},{"name":"thread","value":"19896@LAPTOP-14OH4E5T.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"deleteUserTest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"There is no such user expected [result: null] but found [{\"result\":null,\"id\":1416806551,\"jsonrpc\":\"2.0\"}]","trace":"java.lang.AssertionError: There is no such user expected [result: null] but found [{\"result\":null,\"id\":1416806551,\"jsonrpc\":\"2.0\"}]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1577)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat api.tests.CreateUserTests.deleteUserTest(CreateUserTests.java:24)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1690227402723,"stop":1690227402778}